src/tranche-space.rain

```diff
@@ -481,7 +514,7 @@ deployments:
   amount-available: decimal18-mul(tranche-total-size tranche-space-available),
   amount: if(amount-is-output amount-available decimal18-div(amount-available final-io-ratio)),
   io-ratio: final-io-ratio,
-  :call<'plottables>(amount io-ratio);
+  :call<'plottables>(tranche-io-ratio amount io-ratio);
```

Comment:
we should show the multiplier

Reply:
sorry more context please, `tranche-io-ratio` is the multiplier what is it that needs to be shown ?

Reply:
Added the multiplier metric.